working/0047. Spread Collections/spread-collections.md

```diff
@@ -422,9 +422,9 @@ var after = [1, more..., 3];
 
 Putting it before has these advantages (some of which are marginal or dubious):
 
-*   It's what JavaScript does. If we assume most people coming to Dart and
-    familiar with a spread operator learned it from JS, that makes it the most
-    familiar syntax.
+*   It's what JavaScript does. If we assume most people coming to Dart learned
+    and presently know the spread operator through JS, that syntax is the most
```

Comment:
@munificent This slightly changes the sense here (from "coming to Dart && familiar with spread => learned it from JS" to "coming to Dart => learned and know from JS"). Are you happy with this change?  Otherwise everything LGTM, thanks for the fixes!

Reply:
Yeah, I'm not sure about this change. The "presently" is kind of confusing. @SebastiaanYN, what didn't you like about the original text?

Reply:
The wording is a bit strange in the original text, which is why I decided to see if there was a better way to word it. There's not much wrong with the original, so keeping it the same is fine.